### PR TITLE
Components

### PR DESCRIPTION
**Pull request by emersonlaurentino**

### CODE REVIEW

**Comment by raphamorim** [comment]
@emersonlaurentino can you rebase it?

File: README.md [hunk 1]
```diff
@@ -157,63 +264,115 @@ import { withRubberBand } from 'react-motions'
...
+```jsx
+import { withWobble } from 'react-motions'
+
+const ComponentWithWobble = withWobble(<div>Wobble!</div>)
+```
 
 # Roadmap
 
 - [ ] `withSequence`
 - [ ] `compose`
```
Comment (felippenardi) [inline README.md:364]: Isn't both `withSquence` and `compose` done?

Comment (raphamorim) [reply README.md:364]: nopes, He actually only worked on Components

Comment (emersonlaurentino) [reply README.md:364]: No, I did not do these, u can?

Comment (felippenardi) [reply README.md:364]: You are just adding the documentation for how to use them, correct?

Comment (emersonlaurentino) [reply README.md:364]: Yes, I made a change in readme at the request of @raphamorim, though I did not created `withSquence` and `compose`.

**Comment by emersonlaurentino** [comment]
#8

File: README.md (hunk 1 above)
Comment (felippenardi) [reply README.md:364]: @emersonlaurentino @raphamorim I’ll work on both of those

**Pull request closed without merging**